docs/man-openmpi/man3/MPI_Accumulate.3.rst

```diff
@@ -172,4 +172,6 @@ ERRORS
 .. include:: ./ERRORS.rst
 
 .. seealso::
-   :ref:`MPI_Put` :ref:`MPI_Get_accumulate` :ref:`MPI_Reduce`
+   * :ref:`MPI_Put`
+   * :ref:`MPI_Get_accumulate`
+   * :ref:`MPI_Reduce`
```

Comment:
In list-ifying all the See Also's, I don't think I have a *strong* opinion, but I do have an opinion. 😄 

We took our cue from other Linux and MacOS man pages on how to do the See Also's.  For example, `man ls` in MacOS 13.1:

![Screenshot 2023-02-01 at 12 13 41 PM](https://user-images.githubusercontent.com/617316/216113811-84b5ae9e-5287-4aa9-bad8-f3f3d64487e3.png)

And if I `man uptime` on Ubuntu 22.04:

![Screenshot 2023-02-01 at 12 14 15 PM](https://user-images.githubusercontent.com/617316/216114098-e28dbde6-5f58-40f0-ac32-2f8f06e37afb.png)

Meaning: if we want to keep with conventions of other man pages, we should probably add commas to our list of man pages, not bullets.

Reply:
I partially agree ;)

The man-page comma-separated links might be good for a terminal view, but definitely not for a HTML page view. The links gets bunched together. I think this should be fixed via some other means, e.g. by some customization of the `conf.py` file. I'll have a look whether this could be done.